test/subscription-connection.js

```diff
@@ -311,7 +311,7 @@ test('subscription connection handles GQL_START message correctly, when payload.
 
   sc.isReady = true
 
-  await sc.handleMessage(JSON.stringify({
+  sc.handleMessage(JSON.stringify({
```

Comment:
I've removed the async/await because the message was sent after the promise resolution, causing the test to end before all expectations were met. 

Should I dig more into this?

Reply:
It's ok

Reply:
not sure why this behavior would change though because of the graphql bump. do you have an explanation for why it changes @gdorsi ?

Reply:
The reason is this change https://github.com/graphql/graphql-js/commit/39581581acc184301cfbb0ee348d0f4b6a21a99f

The have added an await to the function that throws the validation error.

Reply:
We could fix it with [this](https://github.com/gdorsi/mercurius/commit/1b05dfb0c5a707fbe4e4a586380542d837d1d36d) change to handleMessage but I'm not sure how this would affect the performance.